testplan/cli/commands/writers.py

```diff
@@ -72,7 +72,7 @@ def __call__(self, result: TestReport) -> TestReport:
             pdf_path=self.filename, pdf_style=self.style.value
         )
         exporter.create_pdf(result)
-        self.logger.test_info(f"PDF written to {self.filename}")
+        self.logger.user_info(f"PDF written to {self.filename}")
```

Comment:
Noticed these are still f-strings instead of the lazy formatting. Can you please update while at it?

Reply:
Okay sure